app/Enumerators/ResponseCode.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enumerators;
+
+enum ResponseCode: int
```

Comment:
Here you could hold all response codes (just move 200 and 201 that were in HttpHeaderStatus enum)